packages/webservice/app/build.gradle.kts

```diff
@@ -48,20 +58,18 @@ tasks.withType<KotlinCompile> {
 }
 
 tasks.register<NpmTask>("installUIDependencies") {
-    setArgs(listOf("ci"))
+    setArgs(listOf("install"))
     setExecOverrides(closureOf<ExecSpec> {
         setWorkingDir("../../../")
     })
 }
 
-tasks.register<Exec>("buildUI") {
-    dependsOn("installUIDependencies")
-
-    workingDir("../../..")
-
-    commandLine = listOf("npx", "lerna", "run", "prodbuild", "--scope", "navi-app", "--stream")
+tasks.register<NpmTask>("buildUI") {
+  dependsOn("installUIDependencies")
+  setArgs(listOf("run-script","demostart"))
```

Comment:
Does this start the navi-app via the node asset server?

Reply:
yes, i added demostart as a new script command to the main navi package.json. 

The original `lerna prodbuild` was causing memory errors in multiple machines even when I allocated 16 GB VM. Not sure why.

Reply:
did not happen again. will log a bug if the memory error shows up. resolving for now.